task06_2/task06_2/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task06_2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Дано слово кинематограф");
+            string input = "кинематограф";
+            Console.WriteLine("Первое слово: " + firstConcat(input));
+            Console.WriteLine("Второе слово: " + secondConcat(input));
+        }
+
+        static string firstConcat(string word)
+        {
+            string first = word.Substring(8);
+            string second = word.Substring(7, 1);
+            string third = word.Substring(4, 2);
+            string forth = word.Substring(2, 1);
+
+            var result = first + second + third + forth;
+            return (result);
+        }
+
+        static string secondConcat(string word)
+        {
```

Comment:
Названия методов в C# принято писать с большой буквы
```suggestion
            Console.WriteLine("Первое слово: " + FirstConcat(input));
            Console.WriteLine("Второе слово: " + SecondConcat(input));
        }

        static string FirstConcat(string word)
        {
            string first = word.Substring(8);
            string second = word.Substring(7, 1);
            string third = word.Substring(4, 2);
            string forth = word.Substring(2, 1);

            var result = first + second + third + forth;
            return (result);
        }

        static string SecondConcat(string word)
        {
```